packages/plugin-search/src/Search/ui/LinkToDoc/index.client.tsx

```diff
@@ -24,6 +26,7 @@ export const LinkToDocClient: React.FC = () => {
 
   const href = `${serverURL}${formatAdminURL({
     adminRoute,
+    basePath,
```

Comment:
Instead of threading base path here, we could replace the anchor tag for `next/link`.

Reply:
Done, no longer sending `basePath` to `formatAdminURL`. 

I have kept this line `const basePath = process.env.NEXT_BASE_PATH || false` because we need it for displaying the URL:
<img width="446" alt="Screenshot 2025-02-03 at 12 56 09 PM" src="https://github.com/user-attachments/assets/a30a5dd8-1766-45fb-acdc-58e73a0ab359" />